src/raftstore/store/peer.rs

```diff
@@ -326,7 +326,7 @@ impl Peer {
         }
 
         debug!("propose command with uuid {:?}", cmd.uuid);
-        metric_incr!("raftstore.propose");
+        metric_incr!(&format!("raftstore.{}.propose", self.peer.get_store_id()));
```

Comment:
The method format is kind of slow. Could you maintain a store prefix: raftstore.{store id}